yast2_rdp.pm

```diff
@@ -0,0 +1,194 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST2's RDP module is behaving
+#          correctly and verifying that RDP service has been
+#          successfully set.
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base "y2_module_guitest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+my $repos = {'12-SP1' => 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP1/SLE-12-SP1/',
```

Comment:
this should not be In any test, if isn't yast2-rdp in the product simply not run on this product

Reply:
yes, if we only test yast2-rdp in the default repos, I will adjust the code.